src/test/java/org/jabref/logic/layout/format/AuthorsTest.java

```diff
@@ -10,151 +11,139 @@
 
 public class AuthorsTest {
 
+    private ParamLayoutFormatter authorsLayoutFormatter;
+
+    @BeforeEach
+    void setup() {
+        authorsLayoutFormatter = new Authors();
+    }
```

Comment:
In this case, the code can be much shorter

```suggestion
    private ParamLayoutFormatter authorsLayoutFormatter = new Authors();
```

Reply:
It doesn't seem that you did this change. Do you have the button "Commit suggestion", too?

![grafik](https://user-images.githubusercontent.com/1366654/116828443-f47dce00-ab9e-11eb-8ee2-32290a115ebc.png)

Reply:
Yes, I do have that button, too. But I just didn't get around to applying your suggestions on this pull requests yesterday. So I'll work on it today.